engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -39,7 +41,7 @@ class CromwellApiServiceActor(val workflowManager: ActorRef, config: Config)
   def receive = runRoute(possibleRoutes)
 }
 
-trait CromwellApiService extends HttpService with PerRequestCreator {
+trait CromwellApiService extends HttpService with PerRequestCreator with ServiceRegistry {
```

Comment:
`ServiceRegistry` is a trait holding a reference to the singleton actor ref for the service registry. Maybe the name is confusing :/